Level2/Echai_funding/README.md

```diff
@@ -7,16 +7,49 @@ all recieved funds get listed below for transparancy and past records.
 All can see total amount of funds Website owner holds.
 
 
-# Sample Hardhat Project
+### Project Description
 
-This project demonstrates a basic Hardhat use case. It comes with a sample contract, a test for that contract, and a script that deploys that contract.
+Echai full stack web app using React, ethers.js and hardhat for testing.
+
+A decentralized application using hardhat and ethers.js and will be deployed on netlify server.
+
+Anyone can donate funds to trust with a message attached to it.
+all recieved funds get listed below for transparancy and past records.
+All can see total amount of funds Website owner holds.
+
+#### Text Stack used:
```

Comment:
write like this 
<ul>
<li>
[x] Solidity
<li>
[x] Ethers
</li>
<li>
[x] React
</li>
<li>
[x] HardHat
</li>
<li>
[x] remix
</li>
</ul>